plugins/cliconf/nxos.py

```diff
@@ -213,7 +214,7 @@ def get_config(self, source="running", format="text", flags=None):
             cmd += " ".join(to_list(flags))
         cmd = cmd.strip()
 
-        return self.send_command(cmd)
+        return self.send_command(cmd, use_cache=True)
```

Comment:
So, this is going to be an API change with ansible-base, which means we'll need to update the minimal version of ansible we are going to support.

Additionally, will this feature be backported into 2.10 and 2.9? If now, we likely need to make this feature flag condition in our collection.

Reply:
In order to reduce dependency on ansible-base we can copy the send_command API in network_cli connection plugin which is what the PR https://github.com/ansible-collections/ansible.netcommon/pull/120/files is doing so, in that case, we won't require backport or minimal ansible version dependency to be updated.

Reply:
++ I think that is the right approach, I guess we'll have to remove the bits in ansible/ansible right?

Reply:
Yes, it can be removed, but need to be careful doing so in order to ensure backward compatibility is maintained.